src/legacy/ui/public/field_editor/components/field_format_editor/editors/color/color.js

```diff
@@ -31,7 +31,7 @@ import {
   DefaultFormatEditor
 } from '../default';
 
-import { DEFAULT_COLOR } from '../../../../../../../core_plugins/kibana/common/field_formats/types/color_default';
+import { DEFAULT_COLOR } from '../../../../../../../../plugins/data/common/field_formats/converters/color_default';
```

Comment:
Wrong reference: we can import only from top level of data/public folder

Reply:
Reimported